sympy/stats/crv_types.py

```diff
@@ -2590,14 +2809,32 @@ def pdf(self, x):
         mu, s = self.mean, self.shape
         return exp(-s*(x - mu)**2 / (2*x*mu**2)) * sqrt(s/((2*pi*x**3)))
 
-    def sample(self):
+    def _sample_numpy(self, size):
+        xm, alpha = float(self.xm), float(self.alpha)
+        numpy = import_module('numpy')
+        if numpy:
+            return (numpy.random.pareto(a=alpha, size=1000) + 1) * xm
+        else:
+            raise NotImplementedError('Sampling the Pareto Distribution requires Numpy.')
```

Comment:
I would remove this `if numpy:` statement... just `return ...` without any exception raising.